src/SumatraStartup.cpp

```diff
@@ -796,7 +796,7 @@ int APIENTRY WinMain(_In_ HINSTANCE hInstance, _In_opt_ HINSTANCE hPrevInstance,
 
     retCode = RunMessageLoop();
     SafeCloseHandle(&hMutex);
-    CleanUpThumbnailCache(gFileHistory);

```

Comment:
because now the FILE_HISTORY_MAX_FREQUENT is not needed for displaying thumbnail on window!!! but CleanUp Method cleane by FILE_HISTORY_MAX_FREQUENT. this method must be rewrite with another algorithm if we want to clean up thumbnails.